sdcflows/utils/epimanip.py

```diff
@@ -188,10 +188,14 @@ def get_trt(in_meta, in_file=None):
             raise ValueError(f"'{trt}'")
 
         return trt
+    elif in_file is None:
+        msg = "Unable to find TotalReadoutTime in metadata and in_file \
+               not defined."
+        raise AssertionError(msg)
```

Comment:
```suggestion
        msg = "`in_file` must be defined if TotalReadoutTime does not appear in `in_meta`."
        raise ValueError(msg)
```

Other changes in this file should be reverted.